stylelint.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     'max-empty-lines': null, // TODO: enable?
     'selector-type-no-unknown': null,
     'function-comma-space-after': null, // TODO: enable?
-    'font-family-no-missing-generic-family-keyword': null, // needed for octicons (no sensible fallback)
+    'font-family-no-missing-generic-family-keyword': null, // needed for Actions (no sensible fallback)
```

Comment:
'font-family-no-missing-generic-family-keyword': null, // needed for Actions (no sensible fallback)